rocky/rocky/views/ooi_detail.py

```diff
@@ -112,6 +112,7 @@ def get_context_data(self, **kwargs):
 
         # TODO: generic solution to render ooi fields properly: https://github.com/minvws/nl-kat-coordination/issues/145
         context["object_details"] = format_display(self.get_ooi_properties(self.ooi), ignore=["json_schema"])
+        context["zombie"] = context["object_details"].pop("zombie", False)
```

Comment:
```suggestion
        context["inactive"] = context["object_details"].pop("inactive", False)
```

Even better would be just to pass the user object (like the suggestion in the OOI add); template logic can then render the correct label